### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by eileenxuan**
Added expected error messages for running as web application. Added related files for solution of Android troubleshooting.

### CODE REVIEW

File: MobileClient/README.md [hunk 1]
```diff
@@ -22,7 +22,19 @@ Any ionic application can be run on a browser for testing basic functionality. T
 ionic serve
 ```
 
-You should be greeted by a registration page in your browser. If this command fails to run, ensure all the prerequisites have been installed correctly. If this step succeeds, you are ready to run on an emulator or device.
+You should be greeted by a registration page in your browser. If this command fails to run, ensure all the prerequisites have been installed correctly. 
+
+If you click scan in the registration page, you will get error:
```
Comment (pbrasil) [inline MobileClient/README.md]:
```suggestion
If you click _scan_ in the registration page, you will get the following error:
```

File: MobileClient/README.md [hunk 2]
```diff
@@ -22,7 +22,19 @@ Any ionic application can be run on a browser for testing basic functionality. T
...
...host:8100/build/main.js:371:13) at t.prototype.invoke (http://localhost:8100/build/polyfills.js:3:9008) at onInvoke (http://localhost:8100/build/vendor.js:30921:21) at t.prototype.invoke (http://localhost:8100/build/polyfills.js:3:9008) at e.prototype.run (http://localhost:8100/build/polyfills.js:3:6455) at Anonymous function (http://localhost:8100/build/polyfills.js:3:4574) at t.prototype.invokeTask (http://localhost:8100/build/polyfills.js:3:9614) at onInvokeTask (http://localhost:8100/build/vendor.js:30912:21) at t.prototype.invokeTask (http://localhost:8100/build/polyfills.js:3:9614) at e.prototype.runTask (http://localhost:8100/build/polyfills.js:3:7057) at s (http://localhost:8100/build/polyfills.js:3:4205) at Anonymous function (http://localhost:8100/build/polyfills.js:3:4612) at t.prototype.invokeTask (http://localhost:8100/build/polyfills.js:3:9614) at onInvokeTask (http://localhost:8100/build/vendor.js:30912:21) at t.prototype.invokeTask (http://localhost:8100/build/polyfills.js:3:9614) at e.prototype.runTask (http://localhost:8100/build/polyfills.js:3:7057) at i (http://localhost:8100/build/polyfills.js:3:3664) at invoke (http://localhost:8100/build/polyfills.js:3:10870)
```
Comment (pbrasil) [inline MobileClient/README.md]: Maybe quote this part?

File: MobileClient/README.md [hunk 3]
```diff
@@ -22,7 +22,19 @@ Any ionic application can be run on a browser for testing basic functionality. T
...
+
+If you click dismiss in the registration page, you will get error:
```
Comment (pbrasil) [inline MobileClient/README.md]:
```suggestion
If you click _dismiss_ in the registration page, you will get the following error:
```

File: MobileClient/README.md [hunk 4]
```diff
@@ -22,7 +22,19 @@ Any ionic application can be run on a browser for testing basic functionality. T
...
+
+Error: Error in ./RegistrationPage class RegistrationPage - inline template:40:2 caused by: Object doesn't support property or method 'dismiss' at DebugAppView.prototype._rethrowWithContext (http://localhost:8100/build/vendor.js:89149:17) at Anonymous function (http://localhost:8100/build/vendor.js:89162:17) at Anonymous function (http://localhost:8100/build/vendor.js:35817:9) at t.prototype.invokeTask (http://localhost:8100/build/polyfills.js:3:9614) at onInvokeTask (http://localhost:8100/build/vendor.js:30912:21) at t.prototype.invokeTask (http://localhost:8100/build/polyfills.js:3:9614) at e.prototype.runTask (http://localhost:8100/build/polyfills.js:3:7057) at invoke (http://localhost:8100/build/polyfills.js:3:10827)
```
Comment (pbrasil) [inline MobileClient/README.md]: Same as before, maybe quote this?

File: MobileClient/README.md [hunk 5]
```diff
@@ -22,7 +22,19 @@ Any ionic application can be run on a browser for testing basic functionality. T
...
+
+Those errors are expected.
```
Comment (pbrasil) [inline MobileClient/README.md]: Italic AND bold :)
We gotta make sure people see this.

File: MobileClient/README.md [hunk 6]
```diff
@@ -71,10 +83,13 @@ ionic cordova platform remove android
 ionic cordova platform add android@6.4.0
 ```
 
+If you get errors like "Error occurred during initialization of VM. Could not reserve enough space for 2097152KB object heap", make sure your jvm is in the correct architecture.
```
Comment (pbrasil) [inline MobileClient/README.md]: Add between parenthesis what is the correct architecture or how to check this.

Comment (eileenxuan) [reply MobileClient/README.md]:
```suggestion
For Windows, if you get errors like "Error occurred during initialization of VM. Could not reserve enough space for 2097152KB object heap", make sure your jvm is in the correct architecture. You can check your _system type_ in System Information APP. For example, you may see _x64-based PC_ for your system type. Then you will need to download _Windows x64 Java SE Development Kit_ from [JDK](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html).
```

**Pull request merged**